third-party/macosx-libevent-config.h

```diff
@@ -10,17 +10,26 @@
 /* Define if libevent should not be compiled with thread support */
 /* #undef DISABLE_THREAD_SUPPORT */
 
+/* Define to 1 if you have the `accept4' function. */
+/* #undef HAVE_ACCEPT4 */
+
+/* Define to 1 if you have the <afunix.h> header file. */
+/* #undef HAVE_AFUNIX_H */
+
 /* Define to 1 if you have the `arc4random' function. */
 #define HAVE_ARC4RANDOM 1
 
+/* Define to 1 if you have the `arc4random_addrandom' function. */
+#define HAVE_ARC4RANDOM_ADDRANDOM 1
+
 /* Define to 1 if you have the `arc4random_buf' function. */
-/* #undef HAVE_ARC4RANDOM_BUF */
+#define HAVE_ARC4RANDOM_BUF 1
 
 /* Define to 1 if you have the <arpa/inet.h> header file. */
 #define HAVE_ARPA_INET_H 1
 
 /* Define to 1 if you have the `clock_gettime' function. */
-/* #undef HAVE_CLOCK_GETTIME */
+#define HAVE_CLOCK_GETTIME 1
```

Comment:
We currently have MACOSX_DEPLOYMENT_TARGET set to 10.10 while this function is available since 10.12 (see e.g. http://manpagez.com/man/3/clock_gettime/). Didn't check all the other definitions, but we need to either regenerate these config files using 10.10 SDK to avoid breaking support for old systems or raise the target version (but still use the appropriate SDK to generate these files, or at least test that it work on what we target).

Reply:
Is this possible to generate configuration for 10.10 without real system? I can setup VM, but maybe there's a more convinient way.

Reply:
Having a 10.10 SDK would be enough. The only issue is, SDK is never distributed separately but is part of Xcode package. Fortunately, there're projects like e.g. https://github.com/devernay/xcodelegacy and https://github.com/phracker/MacOSX-SDKs.